### PR TITLE
Debate: Autoclose inactive issues after X days

### PR DESCRIPTION
**Pull request by Cyberboss**
Currently set to a year, up for debate. Any activity will prevent closure (that means any user comment). This is mainly to help trim down the issues list of the crap that no one will ever bother to look at again. If people care enough, they can comment to prevent a closure. Currently limited to actioning on 1 issue per hour to prevent the massive influx of closure notifications it'll cause.

@tgstation/commit-access 

If we decide to do this @KorPhaeron has to install it here: https://github.com/apps/stale

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 3
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - BYOND Issue
+  - Priority: High
+  - Priority: Critical
+  - Maintainability/Hinders improvements 
+  - Performance
```
[the remark this answers is not in the document]
Comment (Jalleo) [reply .github/stale.yml]: I would personally like certain bugs which are from the same object to be merged together.

**Comment by chesse20** [comment]
nope no support for this

**Comment by ninjanomnom** [comment]
Can this also be set to ignore issues in a milestone or project?

**Comment by Cyberboss** [comment]
@ninjanomnom Already done

**Comment by DaxDupont** [comment]
I've been trying to shill this idea for months now

**Comment by ShizCalev** [comment]
What exactly does this solve though beside artificially reducing the issue count? This issues would still exist, you'd just have to go through two more steps to get to them to fix things.

**Comment by KorPhaeron** [comment]
I checked issues I personally opened and I have valid ones from 2016/2017

**Comment by DaxDupont** [comment]
Problem with older issues is that they often are ignored by even the most dedicated of coders.
When I see new issues pop up I generally fix them if I know how, I don't go past the first few pages.

Old issues get rediscovered and potentially fixed instead of remaining stale till someone bothers to go through them. Churning issues is probably a better  idea.

**Comment by AnturK** [comment]
I wouldn't mind this for needs reproducing / not a bug issues but for the rest just feels like sweeping the bugs under the rug.

**Comment by fludd12** [comment]
Some of the issues in the tracker are for code that literally no longer exists. I support this. If you have an issue thats still relevant, just do a bump every once in a while to remind people it exists.

**Comment by BeeSting12** [comment]
Closing an old issue doesn't make it disappear. If you really want to close it then how about fix the problem instead?

**Comment by Mickyan** [comment]
Issues that are a year+ old with no activity are unlikely to be looked at again, let alone purposefully fixed or even reproduced; a list of bugs stops being useful when it's so bloated and outdated

Besides we still get duplicated issues all the time as it is - if a bug is still relevant after it gets closed one year later it will be reported again no doubt

**Comment by Dorsisdwarf** [comment]
I'd like to point out that at 1 per hour, the autocloser will spend something like a month and a half posting every half an hour 24/7. Wouldnt it make more sense to just disable the notification bots for 5 minutes and make it do all of them at once?

**Comment by Jalleo** [comment]
most of the bugs right at the back are probably still relevant. A proper way to resolve this is to probably use the milestone or projects system (or another one of some kind) to make it easier to work out which ones are fixed when certain things are changed or caused by actually making it easier to look through them rather than going through a lot of pages of issues to locate the issues each time)

**Pull request closed without merging**